<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jianjie</class>
 <widget class="QWidget" name="jianjie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园寻路系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="School.qrc">
    <normaloff>:/School/Picture/tb.png</normaloff>:/School/Picture/tb.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/School/Picture/jianjie.png);</string>
  </property>
  <widget class="QLabel" name="MC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="Jj_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入简介：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qd_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>327</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fh_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>327</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="School.qrc"/>
 </resources>
 <connections/>
</ui>
